.size-limit.js

```diff
@@ -13,7 +13,7 @@ module.exports = [
     name: 'All Integrations - CDN',
     path: 'dist/legacy/js-integrations/*.min.js',
     gzip: true,
-    limit: '415.3 kB',
+    limit: '431.445 kB',
```

Comment:
It's also important to note that this integration SDK's size is bumped by more than 15 kB (minified). This size change is significant to leave an impact.
You might want to go back and investigate the cause. Perhaps the storage module import?

Reply:
Yes this is due to Storage module import